src/rdkafka_broker.c

```diff
@@ -2479,6 +2529,20 @@ static int rd_kafka_broker_cmp_by_id (const void *_a, const void *_b) {
 }
 
 
+/**
+ * @brief Set the broker logname (used in logs) to a copy of \p logname.
+ *
+ * @locality any
+ * @locks none
+ */
```

Comment:
what does locks none mean, given you're locking `rkb->rkb_logname_lock`

Reply:
`@locks` instructs the caller what locks it needs to hold to call this function.

Reply:
TIL